ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py

```diff
@@ -822,3 +822,49 @@ def test_digamma(
         on_device=on_device,
         x=x[0],
     )
+
+
+@st.composite
+def _sparsify_tensor_stg(draw):
+    dtype, tensor, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            ret_shape=True,
+            min_num_dims=1,
+            min_dim_size=10,
```

Comment:
```suggestion
            min_dim_size=1,
```
No need to have such large dim_size.